simulation/model.py

```diff
@@ -123,7 +121,7 @@ def create_and_run_simulation(
     annual_renovation_rate: float,
     household_num_lookahead_years: int,
     heating_system_hassle_factor: float,
-    intervention: str,
+    interventions: List[InterventionType],
```

Comment:
I think this is technically true, but don't think it matters too much.
```suggestion
    interventions: Optional[List[InterventionType]],
```